File: system-test/sec-tests/python/sec_19_ssrf.py
# SEC-19: SSRF — Server-Side Request Forgery
# Test: TC-CR-026
import requests
from flask import Flask, request

app = Flask(__name__)

@app.route('/fetch')
def fetch_url():
    url = request.args.get('url')
    response = requests.get(url)
    return response.content

File: system-test/sec-tests/javascript/sec_19_ssrf.js
// SEC-19: SSRF — Server-Side Request Forgery
// Test: TC-CR-026
const express = require('express');
const axios = require('axios');
const app = express();

app.get('/fetch', async (req, res) => {
  const url = req.query.url;
  const response = await axios.get(url);  // SSRF — user-controlled URL
  res.send(response.data);
});
